README.md

```diff
@@ -137,113 +155,139 @@ cp ../*.deb ~/debs/xenial
 
 The deb will be available into the `~/debs/xenial` directory.
 
-### From source
+## Usage
 
-Source-based installation is not recommended for production usage, but is very
-handy for testing or development purpose.
-Get the source by cloning this repo and do a pip install.
+### Configuration
 
-As pip will have to copy files to /etc/cloud-info-provider directory, the
-installation user should be able to write to it, so it is recommended to create
-it before using pip.
+#### Static information
 
-```sh
-sudo mkdir /etc/cloud-info-provider
-sudo chgrp you_user /etc/cloud-info-provider
-sudo chmod g+rwx /etc/cloud-info-provider
-```
+The cloud-info-provider uses a YAML file describing the static information of
```

Comment:
Could be appropriate to clarify that a default is available in the repo (and/or link to it).